config/app_config.cfg

```diff
@@ -11,15 +11,15 @@ TAC = 0
 DNS_PRIMARY = 1.1.1.1
 DNS_SECONDARY = 8.8.8.8
 STATIC_IP_POOL = 10.250.251.0/24 
-IPV4_MTU = 1350
+IPV4_MTU = 1450
```

Comment:
Let's put `IPV4_MTU` in the `[GLOBAL]` entry as well, so that future users are aware of this option (like `DNS_PRIMARY`).

Reply:
OK.